api/v1/openapi.yaml

```diff
@@ -2350,11 +2350,6 @@ definitions:
             properties:
               enabled:
                 type: boolean
-          gracefulTermination:
-            type: object
-            properties:
-              enabled:
-                type: boolean
```

Comment:
I think in the past people refrained from dropping entries from the openapi schema, as potentially a breaking change. I'm not sure what's the consensus these days though.

Reply:
Can we just mark it as deprecated?

Reply:
Added to the field description "Deprecated".